core/src/types.js

```diff
@@ -49,6 +49,9 @@ export type EventType = 'INVALID'
     | 'TRANSFER_STATUS_CHANGED'
     | 'BULK_TRANSFER_STATUS_CHANGED';
 
+export type MemberType = 'BUSINESS'
```

Comment:
Does this actually restrict the MemberType to only BUSINESS or LICENSED_TPP in JS? Or it is just informative?

Reply:
it will fail if the call is different value than these. This is not exactly enum, but only way to limit.

Reply:
Consider adding comments explaining what do these types mean